Docs/packer seo functions batch3 (#13284)
hashicorp/packer: website/content/docs/templates/hcl_templates/functions/encoding/yamldecode.mdx

---
page_title: yamldecode function reference
description: |-

>   The `yamldecode` function decodes a YAML string into a representation of its
>   value. Learn how to use the `yamldecode` function in Packer templates.

Reviewer: ```suggestion
  The `yamldecode` function decodes a YAML string into its corresponding Packer value.
```
